api/src/modules/geo-coding/strategies/point-of-production.geocoding.service.ts

```diff
@@ -25,12 +25,22 @@ export class PointOfProductionGeocodingStrategy extends BaseStrategy {
             lng: sourcingData.locationLongitude,
           },
         });
+      let adminRegionId: string;
 
-      const { adminRegionId } =
-        await this.adminRegionService.getClosestAdminRegionByCoordinates({
-          lng: sourcingData.locationLongitude,
-          lat: sourcingData.locationLatitude,
-        });
+      try {
```

Comment:
Check if this logic piece shared by different geocoding strategies can be externalised to the class extended by all